### PR TITLE
feat: add changelog and vulnerability check

### LINKED ISSUE #34: Implement changelog

### PR DESCRIPTION
**Pull request by AlejandroFernandezLuces**
Closes #34 and #35

### CODE REVIEW

**Comment by RobPasMue** [comment]
Looks like you are getting some vulnerabilities. My recommendation is to merge and then fix.

**Comment by RobPasMue** [comment]
You shouldn't have many in any case

File: pyproject.toml [hunk 1]
```diff
@@ -4,7 +4,7 @@ build-backend = "flit_core.buildapi"
 
 [project]
 name = "ansys-tools-visualization-interface"
-version = "0.2.dev0"
+version = "0.2.0"
```
Comment (RobPasMue) [inline pyproject.toml:7]: This should be ``0.3.dev0`` now

**Comment by AlejandroFernandezLuces** [comment]
Can I get approval?

**Review by RobPasMue — APPROVED** [review]

**Pull request merged**